### ISSUE TITLE
Windows Norton Antivirus flags oculante.exe as WS.Reputation.1 and deletes the binary

### ISSUE THREAD

**Issue by eugenesvk** [woelper/oculante#94]
I've +1'ed on the reputation part within the antivirus UI; also uploaded the binary to virustotal.com (no engines detected anything there), not sure how to properly fix it, though (just read a lot of complaints about such false flags)

**Comment by woelper** [comment]
Not sure what I can do about this. The only thing I know is that there is no malicious code in there. It's pretty annoying for open source developers that you have to put a lot of effort into not being deleted right after downloading or have to get some kind of paid developer account. Not sure if there is some way to sign/whitelist the app?

**Comment by eugenesvk** [comment]
Yeah, sorry, don't have a solution for you, I've only seen other open source devs asking users to submit false positives requests to the AV vendors (maybe you could do the same in the readme pointing to the clean results of virustotal?)
`WS.Reputation.1` is about how often your app is seen in the wild, so if the chicken&egg issue is resolved and your app is used more, it should gain more reputation ;) (but yeah, that's annoying :()

**Comment by woelper** [comment]
Not sure what to do about this one. It is also annoying as it breaks auto-update. While I could pay for some kind of certificate, I don't like being forced to do so. Not sure which other options exist to "sign" the executable in some way. Even then, there must be a way to do so in release as I don't want that to be a manual process every time the app is updated.
